zeal-at-point.el

```diff
@@ -130,19 +137,25 @@ the combined docset.")
 
 (defvar zeal-at-point--docset-history nil)
 
-(defvar zeal-at-point-zeal-version
-  (when (executable-find "zeal")
-    (let ((output (with-temp-buffer
-                    (call-process "zeal" nil t nil "--version")
-                    (buffer-string))))
-      (when (string-match "Zeal \\([[:digit:]\\.]+\\)" output)
-        (match-string 1 output))))
-  "The version of zeal installed on the system.")
-
 (unless (fboundp 'setq-local)
   (defmacro setq-local (var val)
     `(set (make-local-variable ',var) ,val)))
 
+(defvar zeal-at-point-zeal-version nil
+  "The version of zeal installed on the system.")
+
+(defun zeal-at-point-get-version ()
+  "Find installed zeal version."
+  (or zeal-at-point-zeal-version
```

Comment:
True, well I could try to add a fix for that if there was any interest in merging the branch (just have another internal variable storing the last known exe location and update version when that changes?).  Otherwise, I can close the pull request.